ts/Data/Formula/FormulaParser.ts

```diff
@@ -444,6 +444,23 @@ function parseArguments(
     return args;
 }
 
+/**
+ * Checks if there's one of the following operator before the negative number
+ * value: '*', '/' or '^'.
+ *
+ * Used to properly indicate a negative value reference or negate a directly
+ * passed number value.
+ */
+function negativeReference(formula: Formula): boolean {
+    const formulaLength = formula.length;
+    const priorFormula = formula[formulaLength - 2];
+    return formula[formulaLength - 1] === '-' && (
+        priorFormula === '*' ||
+        priorFormula === '/' ||
+        priorFormula === '^'
```

Comment:
In my opinion, it would be better to use `priorFormula.match(/\*|\/|\^)` or another regular expression. In the future, this IF will be larger and larger.